src/components/ConcertCard.js

```diff
@@ -2,71 +2,103 @@ import styled from "styled-components/macro";
 import PropTypes from "prop-types";
 import "../GlobalStyles";
 import "../data.json";
+import Button from "./Button";
 
 ConcertCard.propTypes = {
   concerts: PropTypes.arrayOf(
     PropTypes.shape({
-      concertDate: PropTypes.string,
-      artistName: PropTypes.string,
-      concertLocation: PropTypes.string,
+      date: PropTypes.string,
+      artist: PropTypes.string,
+      location: PropTypes.string,
       id: PropTypes.string,
+      image: PropTypes.node,
+      price: PropTypes.string,
     })
   ),
+  onClick: PropTypes.func,
 };
 
 export default function ConcertCard({
-  concertDate,
-  artistName,
-  concertLocation,
+  date,
+  artist,
+  location,
+  image,
+  onClick,
 }) {
-  const createDate = concertDate.split(", ");
+  const createDate = date.split(", ");
   const Date = [createDate[1].replace(",", " ")];
 
-  const createLocation = concertLocation.split(",");
+  const createLocation = location.split(",");
   const Location = [createLocation[1].replace(",", " ")];
 
   return (
-    <Wrapper>
-      <DateStyle>{Date}</DateStyle>
-      <ArtistStyle>{artistName}</ArtistStyle>
-      <LocationStyle>{Location}</LocationStyle>
+    <Wrapper onClick={onClick}>
+      <TextStyle>
+        <DateStyle>{Date}</DateStyle>
+        <ArtistStyle>{artist}</ArtistStyle>
+        <LocationStyle>{Location}</LocationStyle>
+      </TextStyle>
+      <Image src={image} alt="" />
     </Wrapper>
```

Comment:
Your `Wrapper` is a `button` element and it shouldn't be,[ according to mdn](https://developer.mozilla.org/en-US/docs/Web/HTML/Inline_elements#conceptual_differences):

> Generally, inline elements may contain only data and other inline elements. You can't put block elements inside inline elements.

I would suggest a section as `Wrapper` and adding the button within the section. Also, the way you implemented it now, it is not really accessible. Something like this:
```suggestion
    <Wrapper>
      <TextStyle>
        <DateStyle>{Date}</DateStyle>
        <ArtistStyle>{artist}</ArtistStyle>
        <LocationStyle>{Location}</LocationStyle>
      </TextStyle>
      <Image src={image} alt="" />
      <button onClick={onClick}>Show more details</button>
    </Wrapper>
```

Reply:
I´d like to don´t use an extra button 🙈, because I think it´s a usual user behavior to just click on the result of a search to get more infos ☺️ So now I define it as a section but keep the onClick in line 35. What do you think about this way?

Reply:
It's okay to do it like that but then again, it's not really accessible. But it would be a better way than using a `button` component 😉 
To check if that really is the user behavior, you could add this to the quality-assurance without telling that people have to click on it 😄